src/pages/basic_data/app_admin/central_import/CentralImportDuplicateSearch.tsx

```diff
@@ -0,0 +1,59 @@
+import { useTranslation } from 'react-i18next';
+import { useFetch } from '../../../../utils/hooks/useFetch';
+import { Registration } from '../../../../types/registration.types';
+import { SearchApiPath } from '../../../../api/apiPaths';
+import { SearchResponse } from '../../../../types/common.types';
+import { ListSkeleton } from '../../../../components/ListSkeleton';
+import { Divider, List, Typography } from '@mui/material';
+import { CentralImportResultItem } from './CentralImportResultItem';
+import { ResourceFieldNames } from '../../../../types/publicationFieldNames';
+import { createSearchQuery, ExpressionStatement, SearchConfig } from '../../../../utils/searchHelpers';
+import { ROWS_PER_PAGE_OPTIONS } from '../../../../utils/constants';
+
+interface CentralImportDuplicateSearchProps {
+  publication: Registration;
+}
+export const CentralImportDuplicateSearch = ({ publication }: CentralImportDuplicateSearchProps) => {
+  const { t } = useTranslation('basicData');
+
+  const searchConfig: SearchConfig = publication.entityDescription?.reference?.doi
+    ? {
+        properties: [
+          {
+            fieldName: ResourceFieldNames.Doi,
+            value: publication.entityDescription.reference.doi,
+            operator: ExpressionStatement.Contains,
+          },
+        ],
+      }
+    : {};
+
+  const searchQuery = createSearchQuery(searchConfig);
+  const numberOfResults = ROWS_PER_PAGE_OPTIONS[1];
+  const [searchResults, isLoadingSearchResults] = useFetch<SearchResponse<Registration>>({
```

Comment:
'useFetch' virker litt forvirrende når man bruker axios i bunn. Men jeg har gjort det samme i mange prosjekter.

Reply:
Jeg tenkte egentlig mest på bruken av ordet "fetch". Fetch er et  api definert av W3C, mens Axios er et bibliotek som baserer seg på XMLHttpRequest.

Reply:
ah sånn ja. personlig tenker jeg her på "fetch" som i verbet, at vi bare skal hente noe uansett hva som skjer bak. men skjønner hva du mener. 
...personlig syns jeg egentlig axois stort sett er overflødig, og at det holder å bruke vanilla `fetch()` :brannfakkel:

Reply:
Grunnen til at vi begynte med axios i utgangspunktet var muligheten for å intercepte og returnere mock-data for apiene. Og så slipper man selvsagt doble promises ;-)